web/olga/charts/static/charts/slider.js

```diff
@@ -0,0 +1,97 @@
+var handle;
+var loop_slider_timeout = null;
+
+$(document).ready(function() {
+    handle = $("#custom-handle");
+
+    $( "#slider" ).slider({
+        value: months_keys_sorted.length - 1,
+        min: 0,
+        max: months_keys_sorted.length - 1,
+        step: 1,
+        slide: function(event, ui) {
+            update_selected_date(ui.value);
+        },
+        create: function() {
+            update_selected_date($(this).slider("value"));
+        }
+    });
+    $('#custom-handle').focus();
+
+    $('#play_retrospective').on('click', function() {
+        if (loop_slider_timeout) {
+            loop_slider_timeout = clearTimeout(loop_slider_timeout);
+            $(this).addClass('btn-default');
+            $(this).removeClass('btn-success');
+        } else {
+            loop_slider_timeout = setTimeout(loop_slider, 1000);
+            $(this).addClass('btn-success');
+            $(this).removeClass('btn-default');
+        }
+
+        return false;
+    });
+
+    prepare_months_table();
+    select_month(months_keys_sorted[months_keys_sorted.length - 1]);
+});
+
+function make_tr_from_tabular_countries(month_key, countries) {
+    countries_trs = '';
```

Comment:
Does `countries_trs` variable?

Reply:
Yes, it is. Added an explicit variable definition into the function body.